quarkus-extension/simple-rest-example/README.md

```diff
@@ -0,0 +1,21 @@
+# Camunda Platform Runtime Quarkus Example
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+
+## Packaging and running the application
+
+The application can be packaged using:
+```shell script
+mvn clean package
+```
+It produces the `quarkus-run.jar` file in the `target/quarkus-app/` directory.
+Be aware that it’s not an _über-jar_ as the dependencies are copied into the `target/quarkus-app/lib/` directory.
+
+If you want to build an _über-jar_, execute the following command:
+```shell script
+mvn clean package -Dquarkus.package.type=uber-jar
+```
+
+The application is now runnable using `java -jar target/quarkus-app/quarkus-run.jar`.
```

Comment:
We can also note the REST API endpoints that are created. Maybe:

```suggestion
The application is now runnable using `java -jar target/quarkus-app/quarkus-run.jar`.

Through your browser, you can:
1. Go to http://localhost:8080/deploy-process to deploy the provided BPMN process.
2. Go to http://localhost:8080/start-process to start a process instance of the deployed BPMN process.


```

Reply:
👍